scripts/resource/basic_discovery.rc

```diff
@@ -46,9 +46,7 @@ else
 end
 
 # Test and see if we have a database connected
-begin
-	framework.db.hosts
```

Comment:
I believe in the past this raised a `::ActiveRecord::ConnectionNotEstablished` exception when the db wasn't connected, but added database proxy these days returns `nil` instead